source/newcommitter.md

```diff
@@ -159,118 +167,136 @@ Other notes about the process are available on the main [Apache site](https://ww
+
     ------------------------------------------------------------
     To: private@[PROJECT].apache.org
     Subject: [VOTE] New committer: Jo Bloggs
-    
+
     [ add the reasons behind your nomination here ]
-    
+
     Voting ends one week from today, i.e. midnight UTC on YYYY-MM-DD
     https://www.timeanddate.com/counters/customcounter.html?year=YYYY&month=MM&day=DD
-    
+
     See voting guidelines at
     https://community.apache.org/newcommitter.html
-    
+
     ------------------------------------------------------------
 
-### PMC Vote Template
-This is the email to commence a vote for a new **PMC candidate**. New PMC members need
-to be voted for by the existing PMC members and subsequently approved by the Board
-(or Incubator PMC for incubating projects).
-    
+### Committer and PMC member candidate Vote Template
+This is the email to commence a vote for new committer that is also candidate for a PMC member.
+
     ------------------------------------------------------------
     To: private@[PROJECT].apache.org
-    Subject: [VOTE] New PMC candidate: Jo Bloggs
-    
+    Subject: [VOTE] New Committer and PMC member candidate: Jo Bloggs
+
     [ add the reasons behind your nomination here ]
-    
+
     Voting ends one week from today, i.e. midnight UTC on YYYY-MM-DD
     https://www.timeanddate.com/counters/customcounter.html?year=YYYY&month=MM&day=DD
-    
+
     See voting guidelines at
     https://community.apache.org/newcommitter.html
-    
-### Close Vote
+
+### Committer Close Vote
+This email ends the vote and reports the result to the project.
+
+    ------------------------------------------------------------
+    To: private@[PROJECT].a.o
+    Subject: [RESULT] [VOTE] New committer: Jo Bloggs
+
+    The vote has now closed. The results are:
+
+    Binding Votes:
+
+    +1 [TOTAL BINDING +1 VOTES]
+     0 [TOTAL BINDING +0/-0 VOTES]
+    -1 [TOTAL BINDING -1 VOTES]
+
+    The vote is ***successful/not successful***
+
+### Committer and New PMC member candidate close Vote
 This email ends the vote and reports the result to the project.
-    
+
     ------------------------------------------------------------
     To: private@[PROJECT].a.o
-    Subject: [RESULT] [VOTE] New committer (or PMC candidate): Jo Bloggs
-    
+    Subject: [RESULT] [VOTE] New committer and PMC member candidate: Jo Bloggs
+
     The vote has now closed. The results are:
-    
+
     Binding Votes:
-    
+
     +1 [TOTAL BINDING +1 VOTES]
      0 [TOTAL BINDING +0/-0 VOTES]
     -1 [TOTAL BINDING -1 VOTES]
-    
+
     The vote is ***successful/not successful***
 
-### Notify Board of new PMC member 
-See [https://www.apache.org/dev/pmc.html#newpmc][1]
+### Notify Board of new PMC member
+In case the new committer is also a new PMC member, the PMC Chair (or another PMC member)
+should notify the board. See [new PMC Member](https://www.apache.org/dev/pmc.html#newpmcmember)
 
```

Comment:
This is no longer required, as long as the PMC private list has been notified. So lines 235 through 238 can be dropped.

Reply:
Ah. Indeed https://whimsy.apache.org/board/minutes/PMC_Membership_Change_Process.html desribes it . Let me correct it